client/signup/steps/design-type-with-store/bluehost-store/index.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import FormButton from 'components/forms/form-button';
+import StepHeader from 'signup/step-header';
+import Button from 'components/button';
+import { abtest } from 'lib/abtest';
+import { localize } from 'i18n-calypso';
+
+function redirectToPartner() {
+	if ( 'bluehostWithWoo' === abtest( 'signupStoreBenchmarking' ) ) {
+		window.location.href = 'https://www.bluehost.com/web-hosting/signup?flow=woocommerce';
+	} else {
+		window.location.href = 'https://www.bluehost.com/wordpress';
+	}
+}
```

Comment:
Hmm, I suppose we're trying to avoid having the abtest fire during render. I'd still suggest renaming the method, since this is a click handler for a button instead of an involuntary redirect.

To avoid window in this component, we might build a controller route that redirects this instead.
Say we we link to something like `/bluehost/start` and the controller at that route then triggers `abtest( 'signupStoreBenchmarking' )` and redirects appropriately